### ISSUE TITLE
virsh migrate works but returns 1

### ISSUE THREAD

**Issue by insatomcat** [seapath/meta-seapath#22]
I can't seem to be able to make live migration work, even though I enabled it in the VM and setup ssh keys to make the ssh transport work.
After investigating I found out that "virsh migrate" actually seems to work fine, but returns 1 (instead of 0). The VirtualDomain pacemaker resource considers the live migration as failed when rc!=0 ...

The only log I've found that could explain this error code is this:

```
Nov 27 11:36:05 hypervisor2 libvirtd[30168]: Unable to write to '/sys/fs/cgroup/devices/machine/qemu-100-centos7SP1.libvirt-qemu/devices.deny': Operation not permitted
Nov 27 11:36:05 hypervisor2 libvirtd[30168]: Group devices ACL is not accessible, disabling whitelisting
```

Any idea what could be missing ?
Thanks

**Comment by insatomcat** [comment]
problem is not present with libvirt 6.3.0 (yocto gatesgarth)

**Comment by insatomcat** [comment]
closed by #62
